pyproject.toml

```diff
@@ -20,8 +20,8 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
-django = ">=4"
+python = ">=3.10"
+django = "^5.0"
```

Comment:
You probably should leave the Django version as is, version locking like this will drop compatibility with Django 4.